index.html

```diff
@@ -8,62 +8,194 @@
+			  <button class="pjt-button">See Project</button>
+			</div>
+            <div class="project-card">
+			  <span class="pjt-card-header heading">Profesional Art Printing Data</span>
+              <span class="pjt-card-sub-header sub-heading">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</span>
+			  <div >
+				<ul class="stack-list">
+					<li><a href="#">html</a></li>
+					<li><a href="#">bootstrap</a></li>
+					<li><a href="#">Ruby</a></li>
+				</ul>
+			  </div>
+			  <button class="pjt-button">See Project</button>
+			</div>
+            <div class="project-card">
+			  <span class="pjt-card-header heading">Profesional Art Printing Data</span>
+              <span class="pjt-card-sub-header sub-heading">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</span>
+			  <div >
+				<ul class="stack-list">
+					<li><a href="#">html</a></li>
+					<li><a href="#">bootstrap</a></li>
+					<li><a href="#">Ruby</a></li>
+				</ul>
+			  </div>
+			  <button class="pjt-button">See Project</button>
+			</div>
+            <div class="project-card">
+			  <span class="pjt-card-header heading">Profesional Art Printing Data</span>
+              <span class="pjt-card-sub-header sub-heading">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</span>
+			  <div >
+				<ul class="stack-list">
+					<li><a href="#">html</a></li>
+					<li><a href="#">bootstrap</a></li>
+					<li><a href="#">Ruby</a></li>
+				</ul>
+			  </div>
+			  <button class="pjt-button">See Project</button>
+			</div>
+            
+            
+          </div>
+        </section>
+        <section class="about">
+
+		  <div class="about-me">
+		    <div class="heading abt-header">About me</div>
+		    <div class="sub-heading abt-sub-header">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</div>
+		    <button class="multiposts-btn">Get My Resume</button>
+		  </div>
+		  <hr>
+		  <div class="about-merits">
+			<ul class="stacks-container">
 			  <li>
-				<a href="#" class="social-link">
-					<img src="assets/images/angelist 1.png" alt="" class="icon">
-				</a> 
+				<span class="stack one">Languages</span>
+				<div>
+				  <ul class="stack-list-container">
+					<li>Javascript</li>
+					<li>Ruby</li>
+				    <li>Html</li>
+				    <li>Css</li>
+				  </ul>
+				  
+				</div> 
 			  </li>
 			  <li>
-				<a href="https://twitter.com/tksarso" class="social-link">
-					<img src="assets/images/twitterIcon.png" alt="" class="icon">
-				</a>
+				<span class="stack two">Frameworks</span>
+				<div>
+				  <ul class="stack-list-container">
+					<li>bootstrap</li>
+					<li>Ruby on Rails</li>
```

Comment:
Your content appears too squeezed. I suggest you follow the project requirements instruction to match figma design.
<img width="1433" alt="Screenshot 2022-12-02 at 14 39 40" src="https://user-images.githubusercontent.com/48546992/205320511-95aebccc-ad6c-4cbe-b695-646ec5c34a8a.png">
This is your page:
<img width="436" alt="Screenshot 2022-12-02 at 17 15 39" src="https://user-images.githubusercontent.com/48546992/205320754-273d7407-5b2f-456f-8877-871143767748.png">
This is the figma design
<img width="183" alt="Screenshot 2022-12-02 at 17 31 38" src="https://user-images.githubusercontent.com/48546992/205320893-527fecb6-fa42-459f-af8e-28f506bae6d6.png">

Reply:
@Nyame-Wolf fixed